packages/perseus/src/widgets/mock-widgets/score-mock-widget.ts

```diff
@@ -0,0 +1,38 @@
+import KhanAnswerTypes from "../../util/answer-types";
```

Comment:
This will probably end up getting moved, or possible removed, as part of the Server Side Scoring project - which is fine!